c7n/resources/appelb.py

```diff
@@ -485,6 +485,64 @@ def process_alb(self, client, alb):
                 alb['LoadBalancerArn'], e)
 
 
+@AppELB.action_registry.register('modify-attributes')
+class AppELBModifyAttributes(BaseAction):
+    """Modify attributes given by string keys and values
+
+    Exposes the ModifyLoadBalancerAttributes API
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: turn-on-elb-deletion-protection
+                resource: app-elb
+                actions:
+                  - type: modify-attributes
+                    "deletion_protection.enabled": "true"
+                    "idle_timeout.timeout_seconds": 120
+    """
+    schema = {
+        'type': 'object',
+        'additionalProperties': False,
+        'properties': {
+            'type': {
+                'enum': ['modify-attributes']},
+            'access_logs.s3.enabled': {
+                'enum': ['true', 'false']},
+            'access_logs.s3.bucket': {'type': 'string'},
+            'access_logs.s3.prefix': {'type': 'string'},
+            'deletion_protection.enabled': {
+                'enum': ['true', 'false']},
+            'idle_timeout.timeout_seconds': {'type': 'number'},
+            'routing.http.desync_mitigation_mode': {
+                'enum': ['monitor', 'defensive', 'strictest']},
+            'routing.http.drop_invalid_header_fields.enabled': {
+                'enum': ['true', 'false']},
+            'routing.http2.enabled': {
+                'enum': ['true', 'false']},
+            'load_balancing.cross_zone.enabled': {
+                'enum': ['true', 'false']},
+        },
+    }
+    permissions = ("elasticloadbalancing:ModifyLoadBalancerAttributes",)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('elbv2')
+        attrs = self.data.copy()
+        attrs.pop('type')
+        for appelb in resources:
+            client.modify_load_balancer_attributes(
```

Comment:
we should have a self.manager.retry here with an err_ignore_code around non existant load balaners

Reply:
Addressed in 56460fdc0